core/src/avm2/globals/stubs.as

```diff
@@ -5,3 +5,6 @@ include "Object.as"
 include "Boolean.as"
 include "Number.as"
 include "String.as"
+include "Array.as"
```

Comment:
Let's sort this list alphabetically (might also be worth to mention it in a comment).